can not restart dev mode with backup database more than once

The postgres container changes file permissions for the database directory of a backup database from prod.

The dev mode init scripts check the file permissions, which no longer belong to the system user and fails to start;

```
...
Caused by: java.lang.RuntimeException: Directory /working/databases/horreum-db/archive-primary-20231122075209/primary-20231122075209 does not have sufficient permissions to operate!
	at io.hyperfoil.tools.horreum.infra.common.resources.PostgresResource.checkIfDirectoryIsEmtpy(PostgresResource.java:94)
	at io.hyperfoil.tools.horreum.infra.common.resources.PostgresResource.init(PostgresResource.java:54)
	at io.hyperfoil.tools.horreum.infra.common.HorreumResources.startContainers(HorreumResources.java:145)
	at io.hyperfoil.tools.horreum.dev.services.deployment.HorreumDevServicesProcessor.startHorreumContainers(HorreumDevServicesProcessor.java:129)
	... 11 more

```

--- a/infra/horreum-infra-common/src/main/java/io/hyperfoil/tools/horreum/infra/common/resources/PostgresResource.java
+++ b/infra/horreum-infra-common/src/main/java/io/hyperfoil/tools/horreum/infra/common/resources/PostgresResource.java
@@ -90,10 +90,7 @@ public class PostgresResource implements ResourceLifecycleManager {
         if(!dir.isDirectory()) {
             throw new RuntimeException("Directory " + directoryPath + " is not a directory!");
         }
-        if (!dir.canRead() && !dir.canWrite()) {
-            throw new RuntimeException("Directory " + directoryPath + " does not have sufficient permissions to operate!");
-        }
-        if (dir.list() == null || dir.list().length == 0) {
+        if (dir.list().length == 0) {
             throw new RuntimeException("Directory " + directoryPath + " does not contain any files!");
         }
     }

